shared/src/main/scala/mlscript/JSBackend.scala

```diff
@@ -556,64 +556,36 @@ class JSBackend(allowUnresolvedSymbols: Boolean) {
 
   private def translateParents(superFields: Ls[Term], constructorScope: Scope)(implicit scope: Scope): Opt[JSExpr] = {
     val bases = superFields.map { sym => sym match {
-      case App(lhs, _) => S(translateTerm(App(lhs, Tup(Ls())))(constructorScope))
-      case _ => S(translateTerm(sym)(constructorScope))
+      case App(lhs, _) => translateTerm(App(lhs, Tup(Ls())))(constructorScope)
+      case _ => translateTerm(sym)(constructorScope)
     } }
 
+    def translateParent(current: JSExpr, base: JSExpr, forceMixin: Bool): JSExpr = {
```

Comment:
What is `forceMixin` for?

Reply:
For non-first parent classes, they must be mixins or we would get more than one parent classes, which is not allowed in JS. It has been renamed as `mixinOnly`.